lib/node_modules/@stdlib/blas/base/zcopy/src/zcopy.f

```diff
@@ -0,0 +1,94 @@
+!>
+! @license Apache-2.0
+!
+! Copyright (c) 2024 The Stdlib Authors.
+!
+! Licensed under the Apache License, Version 2.0 (the "License");
+! you may not use this file except in compliance with the License.
+! You may obtain a copy of the License at
+!
+!    http://www.apache.org/licenses/LICENSE-2.0
+!
+! Unless required by applicable law or agreed to in writing, software
+! distributed under the License is distributed on an "AS IS" BASIS,
+! WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+! See the License for the specific language governing permissions and
+! limitations under the License.
+!<
+
+!> Copies values from one complex double-precision floating-point vector to another complex double-precision floating-point vector.
+!
+! ## Notes
+!
+! * Modified version of reference BLAS level1 routine (version 3.9.0). Updated to "free form" Fortran 95.
+!
+! ## Authors
+!
+! * Univ. of Tennessee
+! * Univ. of California Berkeley
+! * Univ. of Colorado Denver
+! * NAG Ltd.
+!
+! ## History
+!
+! * Jack Dongarra, linpack, 3/11/78.
```

Comment:
```suggestion
! * Jack Dongarra, linpack, 4/11/78.
```

The extended comment should match what is in netlib: https://github.com/OpenMathLib/OpenBLAS/blob/develop/lapack-netlib/BLAS/SRC/zcopy.f